### ISSUE TITLE
Clear Transcripts?

### ISSUE THREAD

**Issue by bennun** [joshsten/redmine_jchat#21]
Is there a way to clear the chat? - Josh

**Comment by joshsten** [comment]
Yes. There is, but not from within the chat. If you connect to the database you can clear the chats table. I'll try to get around to updating the plugin over the next week with a couple updates.
